10-General.inc.md

```diff
@@ -144,7 +146,7 @@ Note: While DASH only requires the presence of `moof/pssh` in the first CMAF chu
 
 A DASH client needs to recognize encrypted content and activate a suitable [=DRM system=], configuring it to decrypt content. The MPD informs a DASH client of the [=protection scheme=] used to protect content, identifies the [=content keys=] that are used and optionally provides the default [=DRM system configuration=] for a set of [=DRM systems=].
 
-A <dfn>DRM system configuration</dfn> is the complete data set required for a DASH client to activate a single [=DRM system=] and configure it to decrypt content using a single [=content key=]. <b>It is supplied by a combination of XML elements in the MPD and/or [=solution-specific logic and configuration=]</b>. A [=DRM system configuration=] often contains:
+The <dfn>DRM system configuration</dfn> is the complete data set required for a DASH client to activate a single [=DRM system=] and configure it to decrypt content using a single [=content key=]. It is supplied by a combination of XML elements in the MPD and/or [=solution-specific logic and configuration=]. The [=DRM system configuration=] often contains:
```

Comment:
The emphasis here - now removed - was quite valuable, I think.

Reply:
Ok to keep it